glib/Signal.cs

```diff
@@ -185,14 +185,7 @@ static void ClosureDisposedCB (object o, EventArgs args)
 				tref.RemoveSignal (name);
 		}
 
-		static EventHandler closure_disposed_cb;
-		static EventHandler ClosureDisposedHandler {
-			get {
-				if (closure_disposed_cb == null)
-					closure_disposed_cb = new EventHandler (ClosureDisposedCB);
-				return closure_disposed_cb;
-			}
-		}
+		static EventHandler ClosureDisposedHandler = new EventHandler (ClosureDisposedCB);
```

Comment:
I usually just write:
`static EventHandler ClosureDisposedHandler = ClosureDisposedCB;`

But it makes no difference other than to trigger linting rules

Reply:
Note: readonly